src/main/java/core/basesyntax/fruitstoreoperation/SupplyFruitOperation.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.fruitstoreoperation;
+
+import core.basesyntax.Storage;
+import core.basesyntax.model.InputDataModel;
+
+public class SupplyFruitOperation implements FruitStoreOperation {
+    private Storage storage;
+
+    public SupplyFruitOperation() {
```

Comment:
```suggestion
    public SupplyFruitOperation(Storage storage) {
```
this will be the best option